### ISSUE TITLE
mailosaurGetUsageLimits command cannot be used with server-level API key

### ISSUE THREAD

**Issue by devopsix** [mailosaur/cypress-mailosaur#29]
The new [`mailosaurGetUsageLimits`](https://github.com/mailosaur/cypress-mailosaur/blob/master/src/mailosaurCommands.js#L221) does not seem to work with trial accounts:

![image](https://user-images.githubusercontent.com/58369032/143578294-2c8cd77d-fe11-4e12-9779-bbc92afe7a36.png)

![image](https://user-images.githubusercontent.com/58369032/143578487-28d5af70-1547-474f-aff9-47d93a4254b7.png)

![image](https://user-images.githubusercontent.com/58369032/143578528-5dde9a3f-d438-48fe-ad64-7da70d3e552d.png)

**Comment by hippyhorse** [comment]
Hi @devopsix, could you confirm that you're using an 'account-level' API key, rather than a server-specific API key to make this API call? This should work on trial accounts.

More info on key types here: https://mailosaur.com/docs/managing-your-account/api-keys/#what-are-account-level-api-keys

**Comment by devopsix** [comment]
I was able to confirm that the command works as expected when an account-level API key is used. Thanks a lot for pointing that out!

But I am afraid that brings up a new problem. I have to use a server-level API key due to having to comply to a least privilege policy. I have updated the issue title to reflect that.

[What are server API keys?](https://mailosaur.com/docs/managing-your-account/api-keys/#what-are-server-api-keys) perfectly describes the reason behind that: “ This type of API key can only access the data of a single server, and have no ability to list or manage servers on your account.”

How can I grant developers access to the `mailosaurGetUsageLimits` command (which I am absolutely required to due to the way Mailosaur handles excess email) without at the same time granting them account-level privileges?
